pom.xml

```diff
@@ -302,6 +302,7 @@
                         <additionalJOptions combine.children="append">
                             <JOption>-J-Dhttp.agent=maven-javadoc-plugin</JOption>
                         </additionalJOptions>
+                        <additionalOptions>--frames</additionalOptions>
```

Comment:
So this is just for the Java 8 variant of the build. I know that frames have disappeared from the 11 version of JavaDoc.

Reply:
Java 11 still supports frames, but the default is no-frames (https://bugs.openjdk.java.net/browse/JDK-8196202)  So this gives us frames with Java 11. And this delightful warning:
```
[WARNING] Javadoc Warnings
[WARNING] javadoc: warning - You have specified to generate frames, by using the --frames option.
[WARNING] The default is currently to not generate frames and the support for
[WARNING] frames will be removed in a future release.
[WARNING] To suppress this warning, remove the --frames option and avoid the use of frames.
```